temporalcli/commandsmd/commands.md

```diff
@@ -4,173 +4,390 @@ Commands for the Temporal CLI.
+Terminate the Batch job with the provided Job Id. You must provide
+a reason motivating the termination, which is stored with the Service
+for later reference.
+
+```
+temporal batch terminate --job-id=YourJobId --reason=YourTerminationReason
+```
 
-`temporal batch terminate --job-id=MyJobId --reason=JobReason`
+Read more: https://docs.temporal.io/cli/batch#terminate
 
 #### Options
 
 * `--job-id` (string) - The Batch Job Id to terminate. Required.
 * `--reason` (string) - Reason for terminating the Batch Job. Required.
 
-### temporal env: Manage environments.
+### temporal env: Manage environments
+Environments help you create and manage grouped key-value presets for Options.
+They provide easy set-up for separate environments, such as "dev" and 
+"prod" work.
 
-Use the '--env <env name>' option with other commands to point the CLI at a different Temporal Server instance. If --env
-is not passed, the 'default' environment is used.
+For example, you might set an endpoint preset for the `--address` option.
+You won't have to enter it each time you use the Temporal CLI utility. Env 
+presets let you configure distinct setups for Temporal Services and Namespaces.
 
-### temporal env delete: Delete an environment or environment property.
+Environment names compartmentalize each key-value store. Use the environment
+name as part of your assignment, as in this example, which configures the 
+"prod" environment:
 
-`temporal env delete --env environment [-k property]`
+```
+temporal env set prod.namespace production.f45a2  
+temporal env set prod.address production.f45a2.tmprl.cloud:7233  
+temporal env set prod.tls-cert-path /temporal/certs/prod.pem  
+temporal env set prod.tls-key-path /temporal/certs/prod.key
+```
 
-Delete an environment or just a single property:
+Check your "prod" presets with `temporal env get prod`:
 
-`temporal env delete --env prod`
-`temporal env delete --env prod -k tls-cert-path`
+```  
+address production.f45a2.tmprl.cloud:7233  
+namespace production.f45a2  
+tls-cert-path /temporal/certs/prod.pem  
+tls-key-path /temporal/certs/prod.key
+```
 
-If the environment is not specified, the `default` environment is deleted:
+To use the environment with a command, pass `--env` followed by the environment
+name. For example, to list workflows in the "prod" environment:
 
-`temporal env delete -k tls-cert-path`
+```
+$ temporal workflow list --env prod
+```
+
+Specify an active environment by setting the `TEMPORAL_ENV` environment
+variable in your shell. The Temporal CLI uses the "default" environment when a
+specific environment was not named.
+
+Read more: https://docs.temporal.io/cli/env
+
+### temporal env delete: Delete an environment or environment property
+
+Remove an environment entirely or remove a key-value pair within an
+environment:
+
+```
+temporal env delete [environment or property]
+```
+
+For example:
+
+```
+temporal env delete --env prod
+temporal env delete --env prod --key tls-cert-path
+```
```

Comment:
I think I would split these into two separate examples, one for the whole env and one for a property within an env. Then you don't need the generic `[environment or property]` form.

Reply:
Agreed. Changing.